var/spack/repos/builtin/packages/hipfort/package.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Hipfort(CMakePackage):
+    """ Radeon Open Compute Parallel Primitives Library"""
+
+    homepage = "https://github.com/ROCmSoftwarePlatform/hipfort"
+    url      = "https://github.com/ROCmSoftwarePlatform/hipfort/archive/rocm-3.8.0.tar.gz"
+
+    maintainers = ['srekolam', 'arjun-raj-kuppala']
+
+    version('3.8.0', sha256='0132e9949f758dd8b8a462d133b3316101440cd503aa6c53bea9e34e61bbb3cc')
+
+    variant('build_type', default='Release', values=("Release", "Debug"), description='CMake build type')
+
+    depends_on('cmake@3:', type='build')
+    for ver in ['3.8.0']:
+        depends_on('hip@' + ver, type='build', when='@' + ver)
+        depends_on('rocm-device-libs@' + ver, type='build', when='@' + ver)
+        depends_on('comgr@' + ver, type='build', when='@' + ver)
+        depends_on('hsa-rocr-dev@' + ver, type='build', when='@' + ver)
+
+    def setup_build_environment(self, env):
+        env.set('CXX', self.spec['hip'].hipcc)
+
+    def cmake_args(self):
+        spec = self.spec
+
+        args = [
+            '-DCMAKE_MODULE_PATH={0}/cmake'.format(spec['hip'].prefix)
```

Comment:
Out of curiosity, why doesn't it work without setting CMAKE_MODULE_PATH?

Reply:
yes, It will work without CMAKE_MODULE_PATH. I will remove this.